src/sentry/api/event_search.py

```diff
@@ -1050,16 +1051,31 @@ def resolve_field_list(fields, snuba_args, auto_fields=True):
     groupby that can be merged into the result of get_snuba_query_args()
     to build a more complete snuba query based on event search conventions.
     """
-    # If project.name is requested, get the project.id from Snuba so we
-    # can use this to look up the name in Sentry
-    if "project.name" in fields:
```

Comment:
Why are you removing this?

Reply:
Users could still request project.name as it is a valid field.

Reply:
Gonna add this back in.
But does it make sense that if both `project.name` and `project` are in fields, to only return `project.name` since its more specific, and so we only have clickhouse run transform once?

Reply:
As long as the frontend correctly handles you returning less columns than it expects.

Reply:
Should this be back?

Reply:
Aye its here now: https://github.com/getsentry/sentry/pull/16988/files#diff-a01803e2d22a93a3007e372eb9058e97R1089-R1091

-- Er sorry I used the `PROJECT_NAME_ALIAS` instead of just plain old `project.name`